dataverse-webapp/src/main/webapp/dashboard-maximumembargo.xhtml

```diff
@@ -24,6 +24,8 @@
         <ui:param name="showDataverseHeader" value="false"/>
         <ui:define name="body">
             <h:form id="dashboardMaximumEmbargoForm" onkeypress="return event.keyCode != 13">
+            
+                <h1>eksport wyników wyszukiwania</h1>
```

Comment:
Why it is preceded with empty line? Please remove it.
Why it is in Polish language? Why not translation mechanism applied here?